src/configs/env.ts

```diff
@@ -199,6 +199,14 @@ export const SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS =
   process.env.REACT_APP_SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS || 'zambia2020';
```

Comment:
New envs are documented [here](https://github.com/onaio/reveal-frontend/blob/master/docs/env.md) and also added to the [.env.sample](https://github.com/onaio/reveal-frontend/blob/master/.env.sample) file.